### PR TITLE
Remove payment request description regex validation. 

### PR DESCRIPTION
**Pull request by sipsorcery**
Now accepting any chars that can get past the xss check.

### CODE REVIEW

File: packages/components/src/components/ui/CreatePaymentRequestPage/CreatePaymentRequestPage.tsx [hunk 1]
```diff
@@ -848,17 +826,6 @@ const CreatePaymentRequestPage = ({
...
-                                <InputTextAreaField
-                                  label="Description"
-                                  maxLength={140}
-                                  value={description}
-                                  onChange={(e) => setDescription(e.target.value)}
-                                  validation={onValidateDescription}
-                                  enableQuickValidation
-                                />
-                              </div>
-
```
Comment (Publicker) [inline packages/components/src/components/ui/CreatePaymentRequestPage/CreatePaymentRequestPage.tsx:861]: Are we deleting the input altogether?

Comment (sipsorcery) [reply packages/components/src/components/ui/CreatePaymentRequestPage/CreatePaymentRequestPage.tsx]: Whoops, no we're not. Put it back now without the validation.

File: packages/components/src/components/ui/CreatePaymentRequestPage/CreatePaymentRequestPage.tsx [hunk 2]
```diff
@@ -854,7 +833,6 @@ const CreatePaymentRequestPage = ({
                                   maxLength={140}
                                   value={description}
                                   onChange={(e) => setDescription(e.target.value)}
-                                  validation={onValidateDescription}
                                   enableQuickValidation
```
Comment (Publicker) [inline packages/components/src/components/ui/CreatePaymentRequestPage/CreatePaymentRequestPage.tsx:836]: `enableQuickValidation` could be removed as there's no validation now.

Comment (sipsorcery) [reply packages/components/src/components/ui/CreatePaymentRequestPage/CreatePaymentRequestPage.tsx]: Thansk. Done.

**Review by Publicker — APPROVED** [review]

**Review by donalnofrixion — APPROVED** [review]

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/111.

**Pull request merged**